jest/index.d.ts

```diff
@@ -111,6 +115,41 @@ declare namespace jest {
         skip: Describe;
     }
 
+    interface MatcherUtils {
+        readonly isNot: boolean;
+        utils: {
+            readonly EXPECTED_COLOR: string;
+            readonly RECEIVED_COLOR: string;
+            ensureActualIsNumber(actual: any, matcherName?: string): void;
+            ensureExpectedIsNumber(actual: any, matcherName?: string): void;
+            ensureNoExpected(actual: any, matcherName?: string): void;
+            ensureNumbers(actual: any, expected: any, matcherName?: string): void;
+            /** get the type of a value with handling the edge cases like `typeof []` and `typeof null` */
+            getType(value: any): string;
+            matcherHint(matcherName: string, received?: string, expected?: string, options?: { secondArgument?: string, isDirectExpectCall?: boolean }): string;
+            pluralize(word: string, count: number): string;
+            printExpected(value: any): string;
+            printReceived(value: any): string;
+            printWithType(name: string, received: any, print: (value: any) => string): string;
+            stringify(object: {}, maxDepth?: number): string;
+        }
+    }
+
+    interface ExpectExtendMap {
+        [key: string]: (this: MatcherUtils, received: any, actual: any) => { message: () => string, pass: boolean };
+    }
+
+    interface Expect {
+        (actual: any): Matchers;
+        anything(): void;
+        any(cs: any): void;
```

Comment:
What is 'cs'? Class? I don't think 'class' is reserved in type space, but if it is this could just be classType or similar